### DISCUSSION TITLE
Public warps for town settlements/events/etc.

### DISCUSSION THREAD

**Discussion by aisukuma** [Stardust-Labs-MC/StardustMC#44] [Ideas]
What are your thoughts on having a public warp command for town settlements, events, and probably other stuff? There were some discussion on the Stardust Labs discord, but I honestly forgot.

Of course, these warps won't be free (as in you'll need to do a first time payment to have a warp spot for your location).

**Comment by ReiTheGoat01** [comment]
I'm down for public warps as long as they're not cheap (50k maybe) and maybe there is a player requirement for where the warp is.

**Reply by aisukuma** [reply]
For the player requirement, do you mean like there needs to be a certain amount of players for the area?

**Reply by ReiTheGoat01** [reply]
Yeah for places like towns there should be more then 4 so small groups working together aren't using it to free a home slot instead of it being a warp for players to traverse the main settlements. As for builds like arena's then maybe a vote could be held to see if players would actually use it

**Comment by aisukuma** [comment]
Public warps will now be a thing
